census/census.ipynb

```diff
@@ -32,16 +31,18 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "%%time\n",
-    "df = spatial.read_parquet('./data/census2010.parq')\n",
-    "df = df.persist()"
+    "df = dd.read_parquet('./data/census2010.parq/', engine='fastparquet').persist()\n",
+    "\n",
+    "df['race'] = df.race.astype('category').cat.as_known()"
```

Comment:
Why does it specify the engine as fastparquet if it then sets the category?  I thought the only reason we require fastparquet is that it preserves the category, so if we're converting categories anyway should it use pyarrow instead?

Reply:
Specifying the engine was my attempt at forcing it to load the categories but it doesn't seem like they were saved to this file or that no longer works. In either case, should remove the engine now that I know that.